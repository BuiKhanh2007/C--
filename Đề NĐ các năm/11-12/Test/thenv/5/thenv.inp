8
2142535
3245245
5423445
8345234
4142356
4674567
8735124
1452356
