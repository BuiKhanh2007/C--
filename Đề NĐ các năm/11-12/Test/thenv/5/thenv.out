G
A
H
G
E
I
D
H
